tests/test_other.py

```diff
@@ -9738,6 +9738,10 @@ def test_warning_flags(self):
     stderr = self.run_process(cmd + ['-Wno-pthreads-mem-growth', '-s', 'USE_PTHREADS', '-s', 'ALLOW_MEMORY_GROWTH'], stderr=PIPE).stderr
     self.assertNotContained('USE_PTHREADS + ALLOW_MEMORY_GROWTH may run non-wasm code slowly, see https://github.com/WebAssembly/design/issues/1271', stderr)
 
+    # check that PTHREADS and BUILD_AS_WORKER exit with error
+    stderr = self.expect_fail(cmd + ['-s', 'USE_PTHREADS', '-s', 'BUILD_AS_WORKER=1'])
+    self.assertContained('error: USE_PTHREADS + BUILD_AS_WORKER require separate modes that don\'t work together, see https://github.com/emscripten-core/emscripten/issues/8854', stderr)
```

Comment:
Can you move this to `test_worker_api` in `tests/test_browser.py` (this seems to be the only place we test BUILD_AS_WORKER)?

Reply:
That makes sense, it's done.